bundle/manifests/maintenance-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/component: kube-rbac-proxy
```

Comment:
Deprecated
https://github.com/Mellanox/network-operator/issues/1201

Reply:
Dropped it, thanks!